src/store/metadata/stub/content/editions/versions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions",
+    "@type": "hydra:Collection",
+    "title": "Net additions to the housing stock",
+    "versions": [
+        {
+            "@type": [
+              "dcat:Distribution",
+              "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-05T08:59:11Z",
+            "title": "Net additions to the housing stock",
+            "summary": "Net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, England, financial year 2022 - 2023.",
+            "description": "This dataset shows the net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, in England, for financial year 2022 - 2023.\n\n### Coverage \n\nEngland\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities,\nCounties,\nMetropolitan Counties,\nNation (England) \n\n### Source\n\n - England: [Department for Levelling Up, Housing and Communities]",
+            "download_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+          "columns": [
+            {
+                "name": "Period",
```

Comment:
name in all column mapping everywhere should be _snake_case_, like in `apprenticeships-achievements_2023-05.json`
"name": "period"